control/bgp.yaml

```diff
@@ -116,4 +120,65 @@ components:
           format: uint32
           maximum: 3600
           default: 30
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
```

Comment:
so we are introducing convention of "spaces" for better readability?

Reply:
There is no extra space here (checked codebase ) . This shows up due to previous eof getting removed on lhs.